src/translations.rs

```diff
@@ -26,7 +26,16 @@ pub enum TranslationError {
         "'{0}' is not valid ISO 639-1, valid languages include: {valid}",
         valid = Iso639a::languages().join(", ")
     )]
-    InvalidLangauge(String)
+    InvalidLangauge(String),
```

Comment:
typo

```suggestion
    InvalidLanguage(String),
```

Reply:
fixed in https://github.com/FlakySL/translatable.rs/commit/4b3c35c07b0799652836f195a0d3b0a3d5dc33d5